feat(react-scheduler): add CurrentTimeIndicator plugin (#2536)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/guides/current-time-indication.md

# React Scheduler - Current Time Indication

> The React Scheduler can display a current time indicator and shade appointments and cells up to the current time. This feature can be enabled in the day, week, and month views, but the month view supports only the shading.

Reviewer: ```suggestion
The React Scheduler can display a current time indicator and shade appointments and cells up to the current time. This feature can be enabled in the day, week, and month views (the month view supports shading only).
```

## Related Plugins

> The following plugin implements this feature:

Reviewer: Why write an introduction if there is only one plugin?

Reviewer: ```suggestion
The [CurrentTimeIndicator](../reference/current-time-indicator.md) plugin renders the current time indicator and shades that cover appointments and timetable cells up to the current time. 
```

Reviewer: and the description also duplicates information you already provided.

Reviewer 2: This is just what we do [in all articles](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/guides/appointment-tooltip/). I guess we can remove the introduction because the title of this section is explanatory enough, but I'm against giving the info as a sentence because that would break the pattern.

Reviewer: Well, it seem strange to add an introductory sentence when you only have one item and one sentence. That is like saying 
"I'm going to the bank at the following time: 
- 16:00"

Reviewer 2: Agreed.
```suggestion

```

> - [CurrentTimeIndicator](../reference/current-time-indicator.md) - renders the current time indicator and shades that cover appointments and timetable cells up to the current time

Reviewer 2: ```suggestion
- [CurrentTimeIndicator](../reference/current-time-indicator.md) - renders the current time indicator and the shading that covers appointments and timetable cells up to the current time
```

## Basic Usage

> Import the [CurrentTimeIndicator](../reference/current-time-indicator.md) plugin. Its `updateInterval` property specifies how frequently the indicator's position should be updated. Set the `shadePastAppointments` and `shadePastCells` properties to `true` to shade past appointments and timetable cells:

Reviewer 2: ```suggestion
Import the [CurrentTimeIndicator](../reference/current-time-indicator.md) plugin. Its `updateInterval` property specifies how frequently the indicator's position should be updated. Set the `shadePastAppointments` and `shadePastCells` properties to `true` to shade previous appointments and timetable cells:
```
